iphone/cli/commands/_buildModule.js

```diff
@@ -460,151 +459,173 @@ iOSModuleBuilder.prototype.buildModule = function buildModule(next) {
+		if (!this.isFramework) {
+			args.push('-headers');
+			args.push(headerPath);
+		}
 	}
-};
 
-iOSModuleBuilder.prototype.verifyBuildArch = function verifyBuildArch(next) {
-	const args = [ '-info', path.join(this.projectDir, 'build', this.isFramework ? this.moduleIdAsIdentifier + '.framework/' + this.moduleIdAsIdentifier : 'lib' + this.moduleId + '.a') ];
-
-	this.logger.info(__('Verifying universal library'));
-	this.logger.debug(__('Running: %s', (this.xcodeEnv.executables.lipo + ' ' + args.join(' ')).cyan));
+	args.push(
+		'-output',
+		path.join(this.projectDir, 'build', moduleId + '.xcframework')
+	);
 
-	appc.subprocess.run(this.xcodeEnv.executables.lipo, args, function (code, out, err) {
+	this.logger.debug(__('Running: %s', (this.xcodeEnv.executables.xcodebuild + ' ' + args.join(' ')).cyan));
+	appc.subprocess.run(this.xcodeEnv.executables.xcodebuild, args, function (code, out, err) {
 		if (code) {
-			this.logger.error(__('Unable to determine the compiled module\'s architecture (code %s):', code));
+			this.logger.error(__('Failed to generate universal binary (code %s):', code));
 			this.logger.error(err.trim() + '\n');
 			process.exit(1);
 		}
+		next();
+	}.bind(this));
+};
 
-		const manifestArchs = this.manifest.architectures.split(' '),
-			buildArchs    = out.substr(out.lastIndexOf(':') + 1).trim().split(' '),
-			buildDiff     = manifestArchs.filter(function (i) { return buildArchs.indexOf(i) < 0; });
+iOSModuleBuilder.prototype.verifyBuildArch = function verifyBuildArch(next) {
+	this.logger.info(__('Verifying universal library'));
 
-		if (buildArchs.length !== manifestArchs.length || buildDiff.length > 0) {
-			this.logger.error(__('There is discrepancy between the architectures specified in module manifest and compiled binary.'));
-			this.logger.error(__('Architectures in manifest: %s', manifestArchs.join(', ')));
-			this.logger.error(__('Compiled binary architectures: %s', buildArchs.join(', ')));
-			this.logger.error(__('Please update manifest to match module binary architectures.') + '\n');
-			process.exit(1);
-		}
+	const findLib = function (dest) {
+		const moduleId = this.isFramework ? this.moduleIdAsIdentifier  : this.moduleId;
+		const libName = this.isFramework ? this.moduleIdAsIdentifier + '.framework' : 'lib' + this.moduleId + '.a';
+		let lib = path.join(this.projectDir, 'build', moduleId + '.xcframework', dest, libName);
+		this.logger.info(__('Looking for ' + lib));
 
-		if (buildArchs.indexOf('arm64') === -1) {
-			this.logger.warn(__('The module is missing 64-bit support.'));
+		if (!fs.existsSync(lib)) {
+			return new Error(__('Unable to find the built %s library', 'Release-' + dest));
 		}
+		return lib;
+	}.bind(this);
 
-		next();
-	}.bind(this));
+	const buildArchs = [];
+
+	let lib = findLib('ios-armv7_arm64');
+	if (lib instanceof Error) {
+		this.logger.warn(__('The module is missing 64-bit support.'));
+	} else {
+		buildArchs.push('armv7');
+		buildArchs.push('arm64');
+	}
+   
```

Comment:
<!--
  1 failure:  iphone/cli/comman...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: iphone/cli/commands/_buildModule.js;
  Line: 601;
-->
- :no_entry_sign: iphone/cli/commands/_buildModule.js line 601 – Trailing spaces not allowed. (no-trailing-spaces)